web/customize_viewer.css

```diff
@@ -55,3 +184,28 @@
     margin-left: 72px;
   }
 }
+
+/**
+ * Implement our custom icons. Disable all of the default icons, since the
+ * buttons now have inline svg in viewer.html.
+ */
+.toolbarButton::before {
+  content: '' !important;
```

Comment:
I think `display: none !important;` should just be enough to not show the pseudoelement